spec/Appendix B -- Grammar Summary.md

```diff
@@ -343,6 +343,19 @@ UnionTypeExtension :
 - extend union Name Directives[Const]? UnionMemberTypes
 - extend union Name Directives[Const]
 
+IntersectionTypeDefinition : Description? intersection Name Directives[Const]?
+IntersectionMemberTypes?
```

Comment:
so IntersectionMemberTypes element is optional? we can have intersection with empty list of types?

Reply:
It is same as Union it could be extended later. So valid to be empty initially in sdl. When validated, it has to be non/empty if I recall correctly, just like unions. But we do allow an intersection with non empty list of member types and currently empty list of possible types due to conflict.

Just like interfaces without implementing types are allowed. I believe :)